waku/v2/node/wakunode2.nim

```diff
@@ -433,8 +443,11 @@ when isMainModule:
       {.gcsafe.}:
         # TODO: libp2p_pubsub_peers is not public, so we need to make this either
         # public in libp2p or do our own peer counting after all.
-        let
-          totalMessages = total_messages.value
+        var
+          totalMessages = 0.float64
```

Comment:
How come total messages is a float?

Reply:
Metric values are stored as `float64` and `totalMessages` just sums over these `float64` values.